jira_agile_metrics/calculators/ageingwip.py

```diff
@@ -24,19 +24,9 @@ def run(self, today=None):
         cycle_data = self.get_result(CycleTimeCalculator)
         cycle_names = [s['name'] for s in self.settings['cycle']]
 
-        start_column = self.settings['committed_column']
-        end_column = self.settings['final_column']
+        committed_column = self.settings['committed_column']
         done_column = self.settings['done_column']
-        
-        if start_column not in cycle_names:
-            logger.error("Committed column %s does not exist", start_column)
-            return None
-        if end_column not in cycle_names:
-            logger.error("Final column %s does not exist", end_column)
-            return None
-        if done_column not in cycle_names:
-            logger.error("Done column %s does not exist", done_column)
-            return None
```

Comment:
Why remove these checks? They are useful for alerting the user if their configuration is internally inconsistent.

Reply:
These checks are done in `config.py: config_to_options` and I'd argue that it is more intuitive and robust to do such sanity checks once, and once only.  (See the [DRY principle](https://matiasfrndz.ch/2017/02/Dont-Repeat-Yourself-DRY)).